src/shim/device.cpp

```diff
@@ -605,6 +605,9 @@ struct sequence_name
     case xrt_core::query::sequence_name::type::tct_all_column:
       seq_name = "tct_4col.txt";
       break;
+    case xrt_core::query::sequence_name::type::nop:
+      seq_name = "nop.elf";
```

Comment:
@maxzhen I don't understand why we add nop.elf here.
Could you please review this code too?